app/models/allocation.rb

```diff
@@ -29,13 +29,6 @@ def has_places?
     number_of_places.to_i.positive?
   end
 
-  def add_invalid_number_of_places_errors
```

Comment:
This was mistakenly added in https://github.com/DFE-Digital/publish-teacher-training/pull/1142